### ISSUE TITLE
GCP shell differences

### ISSUE THREAD

**Issue by exsilica** [Islandora-Collaboration-Group/ISLE#228]
When using [Google Cloud Platform: ISLE-Documentation](https://docs.google.com/document/d/1r-I9n8mUhlJ5Z-X9AO-RrikhzH6I8bTPIL25LmtRbxE/edit), users encounter some following differences from the GCP Ubuntu shell than what the ISLE documentation expects. I've made a Doc from the old documentation and we are working through the diffs with the updated documentation now. Please streamline this into the Dev site install guide as appropriate: https://docs.google.com/document/d/1CiIdLz5DsZy_RhnJeC_RFpuaPMnYtJl9c2KHEJYvZSE/edit?usp=sharing

**Comment by g7morris** [comment]
Curious if this been adapted into the Cookbook @marksandford or @dwk2 ?

**Comment by dwk2** [comment]
Little has been done with this. The ISLE docs link to this google doc:
Installing
ISLE on Google Cloud Platform
<https://docs.google.com/document/d/1r-I9n8mUhlJ5Z-X9AO-RrikhzH6I8bTPIL25LmtRbxE/edit#>.
(And the supporting doc has not been incorporated into the primary just
mentioned google doc; it also had been put in the trash by its owner, so
I've made this copy here and stored it with the primary doc: Diffs -
Running through Google Cloud platform
<https://docs.google.com/document/d/16SKrWTaTkq6nWYraGu97JsBpfglY2mPErCd_ON3kqnk/edit>.)
My preference is to leave this as a linked google doc and not take the time
to convert it into markdown.

David Keiser-Clark

**Comment by g7morris** [comment]
So is this a vote to close this ticket as solved by this linked Gdoc?

**Comment by dwk2** [comment]
Yes, exactly! Sorry for not being more clear. And thank you for cleaning
out the issues queue.
David

**Comment by dwk2** [comment]
Little has been done with this. The ISLE docs link to this google doc: [Installing ISLE on Google Cloud Platform](https://docs.google.com/document/d/1r-I9n8mUhlJ5Z-X9AO-RrikhzH6I8bTPIL25LmtRbxE/edit#). (And the supporting doc has not been incorporated into the primary just mentioned google doc; it also had been put in the trash by its owner, so I've made this copy here and stored it with the primary doc: [Diffs - Running through Google Cloud platform](https://docs.google.com/document/d/16SKrWTaTkq6nWYraGu97JsBpfglY2mPErCd_ON3kqnk/edit).)  My preference is to leave this as a linked google doc and not take the time to convert it into markdown.

Closing this ticket as it is solved by this linked Gdoc.
